udp_driver/design/README.md

```diff
@@ -1,9 +1,3 @@
-udp_driver
-===============
-
-This is the design document for the `udp_driver` package.
-
-
```

Comment:
Why remove this?

Reply:
honestly i didnt mean to edit the design document. whoops